sycl/source/handler.cpp

```diff
@@ -127,9 +127,8 @@ event handler::finalize() {
   }
 
   unique_ptr_class<detail::CG> CommandGroup;
-  switch (MCGType) {
+  switch (static_cast<detail::CG::CGTYPE>(getType())) {
```

Comment:
Why do you need `static_cast` here? `getType` returns CGTYPE, doesn't it?
```suggestion
  switch (getType()) {
```